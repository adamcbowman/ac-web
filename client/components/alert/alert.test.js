import React from 'react'
import Renderer from 'react-test-renderer'
import { Danger, Info, Warning, Success } from './'

test('alert component', () => {
    ;[Danger, Info, Warning, Success].forEach(Alert => {
        const alert = Renderer.create(<Alert>Content</Alert>)

        expect(alert).toMatchSnapshot()
    })
})
