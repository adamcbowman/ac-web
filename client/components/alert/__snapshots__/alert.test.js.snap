// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert component 1`] = `
<div
  className="Danger"
>
  Content
</div>
`;

exports[`alert component 2`] = `
<div
  className="Info"
>
  Content
</div>
`;

exports[`alert component 3`] = `
<div
  className="Warning"
>
  Content
</div>
`;

exports[`alert component 4`] = `
<div
  className="Success"
>
  Content
</div>
`;
